packages/components/src/search-list-control/index.js

```diff
@@ -176,12 +176,9 @@ export class SearchListControl extends Component {
 		}
 
 		return (
-			<MenuGroup
-				label={ messages.list }
-				className="woocommerce-search-list__list"
-			>
+			<div className="woocommerce-search-list__list">
```

Comment:
Do you think we could make this an `<ul>`? That means inside `renderList()` we will need to wrap `renderItem( ... )` with a `<li>` and it will require some CSS changes.